Makefile

```diff
@@ -7,7 +7,13 @@ images:
 	./hack/images.sh $(DOCKER_REPO_OVERRIDE)
 
 install:
-	./hack/install.sh
+	INSTALL_SERVING="true" INSTALL_EVENTING="true" ./hack/install.sh
```

Comment:
If you go with the suggestion below, you'll only have to override the `false` switches here.